### DISCUSSION TITLE
How change url?

### DISCUSSION THREAD

**Discussion by ImabYa** [srcnalt/OpenAI-Unity#30] [Q&A] [answered]
When I set model to "gpt-3.5-turbo",Unity went error.

**Comment by srcnalt** [comment, marked as answer]
gpt-3.5-turbo can only be used with Chat Completion API, if you entered that to Text Completion request then you will get an error.
In package version v0.1.8 ChatGPT sample switched to Chat Completion you can take a look at there.
